### PR TITLE
[cli] Update the version to 2.0.2

### PR DESCRIPTION
**Pull request by daniele77**
**Describe the pull request**

- #### What does your PR fix?
  Update cli library

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by FrankXie05** [comment]
@daniele77  Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).
Also please run command `vcpkg x-add-version --all` then commit the changes.

**Comment by daniele77** [comment]
> @daniele77 Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports). Also please run command `vcpkg x-add-version --all` then commit the changes.

Done. 

But please notice that the command `vcpkg x-add-version --all` gives errors on the current repository because of errors in other libraries' ports). E.g.;

```
Error: Unable to parse versions file /home/daniele/projects/sideprojects/github/vcpkg/versions/m-/muparser.json.
Error: failed to parse versions file for `muparser`:
$.versions[2] (a version database entry): 'version' text was not a relaxed version:
Error: String `2.3.3-1` must only contain dot-separated numeric values without leading zeroes.
```

So, instead of `vcpkg x-add-version --all` I used `vcpkg x-add-version cli`.

**Comment by FrankXie05** [comment]
@daniele77 The command `-all` will check all your local port `version` fields. 
According to your error, it shows that the version field of your local port `muparser` is obviously not in compliance with the specification.
Of course, it doesn't matter if the content of the locally modified file is not committed. :)

File: ports/cli/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "cli",
-  "version": "2.0.1",
+  "version": "2.0.2",
+  "port-version": 1,
```
Comment (FrankXie05) [inline ports/cli/vcpkg.json:4]:
```suggestion
```

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: versions/baseline.json [hunk 2]
```diff
@@ -1433,8 +1433,8 @@
       "port-version": 5
     },
     "cli": {
-      "baseline": "2.0.1",
-      "port-version": 0
+      "baseline": "2.0.2",
+      "port-version": 1
```
Comment (FrankXie05) [inline versions/baseline.json:1437]:
```suggestion
      "port-version": 0
```

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**